docs/operations.md

```diff
@@ -377,3 +377,11 @@ of the last route table update `X-Timestamp`, using a HEAD request:
     X-Count: 1
     X-Timestamp: 1517777628
     Date: Sun, 04 Feb 2018 20:54:31 GMT
+
+The number of routes given is limited (1024 routes by default).
+In order to control this limits, there are two filters: `limit` and
```

Comment:
what do you think of using the wording "parameter" or "query param" instead of "filter" in order to avoid overloading the word "filter", since it is used very often in skipper for something very different?